pyproject.toml

```diff
@@ -18,7 +18,7 @@ grand-challenge-forge = "grand_challenge_forge.cli:cli"
 
 [tool.poetry.dependencies]
 python = "^3.10"
-cookiecutter = "^2.5.0"
+jinja2 = "^3.1.4"
 click = "*"
 jsonschema = "^4.20.0"
```

Comment:
From experience it's better to use `"*"` dependencies for libraries (unless you're absolutely certain you don't support the next version of a dependency, which you do not know until it is out).

Reply:
Mhh. Good suggestion. Would it not make more sense to use '>='? That way you ensure no-one tries to run it on older versions that might not work? IIRC the difference is that '^' doesn't do major versions.

Reply:
Libraries need to be quite liberal with what they support otherwise you cause problems for downstream users in applications. You have to remember that not everyone uses semantic versioning. In my experience I have found it better to stick to a default of `"*"` and then only add a minimum version if you're 100% certain that it is incompatible.